content/cybersecurity/concepts/phishing/phishing.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Phishing'
+Description: 'Phishing is a form of fraud, which aims to trick users into sharing sensitive information or personal data or downloading malware.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Phishing** is a form of fraud which aims to trick users into sharing sensitive information or personal data or downloading malware. Phishing can happen through many mediums, including emails, text messages, voice messages, phone calls, or social media. Phishing is one of the most common types of [cyber attacks](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack), and it relies on [social engineering](https://www.codecademy.com/resources/docs/cybersecurity/social-engineering) to defraud users.
+
+## Consequences of Phishing
+
+Phishing can harm users in many forms with a wide range of severity. Successful phishing attacks can result in the following:
+
+- Identity theft,
+- Financial loss,
+- Damage to hardware (e.g. laptop) or software,
+- Damage to personal or business reputation.
+
+## Types of Phishing
+
+Phishing can happen through any communication channel, and usually, the attacker pretends to be a person or organization the user recognizes or trusts. The following table contains the different types of phishing techniques:
+
+| Type of Phishing          | Definition |
+| --------------------------| -----------|
+| Bulk Phishing Emails      | Deceptive messages sent to a large number of recipients, impersonating legitimate organizations or people to obtain sensitive information or infecting devices with malware. This is the most commonly used type of phishing technique. |
+| [Spare Phishing](https://www.codecademy.com/resources/docs/cybersecurity/spear-phishing)             | Personalized, fraudulent emails that trick users into revealing sensitive information or performing malicious actions. |
+| Business Email Compromise | Manipulated or compromised legitimate email accounts within a business or organization that tries to deceive employees or customers into sharing sensitive information or making financial transactions. |
+| SMS Phishing              | Phishing through SMS or phone text messages. |
+| Voice Phishing            | Phishing through phone calls or voice messages. |
+| Social Media Phishing     | Impersonated or compromised accounts of individuals or organizations on social media platforms that try to trick users into revealing personal information or manipulating them into downloading harmful content. |
+| App or In-App Phishing    | Phishing that occurs within a mobile or web application through deceptive fraudulent interfaces. |
+
+## Signs of Phishing
+
+Although phishing emails or messages can be very decisive, certain characteristics of phishing can help identify fraudulent content. These characteristics are:
```

Comment:
"decisive" feels a little strange here to my ear. Is the intention to convey that the email can seem direct and authoritative? Or that they can take many forms, there's a diversity of campaigns or approaches, however, they often share these signatures/characteristics?

Reply:
Maybe "decisive" is not the right word. I meant something like "difficult to recognize" or "can be very convincing". I rephrased the sentence without "decisive".